packages/ui/package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "@ballerine/ui",
+  "private": true,
+  "version": "0.0.1",
+  "type": "module",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "exports": {
+    ".": {
+      "import": "./lib/index.js",
+      "types": "./lib/index.d.ts"
+    },
+    "./components/*": "./lib/components/*",
+    "./lib/style.css": "./lib/style.css"
+  },
+  "scripts": {
+    "dev": "vite build --watch",
```

Comment:
Is `"dev": "vite build --watch"` supposed to be `"dev": "vite"`?

Reply:
`vite` won't produce any output in `lib` which is may be needed for runtime development
Though i think dev vite config could be added